doc/how-to/getting_started_tcm.rst

```diff
@@ -0,0 +1,398 @@
+*   The Tarantool Enterprise Edition SDK 3.0 or later in the ``tar.gz`` archive.
+    See :ref:`enterprise-setup` for information about getting the archive.
+
+For more detailed information about using |tcm|, refer to :ref:`tcm`.
+
+..  _getting_started_tcm_setup:
+
+Setting up Tarantool EE
+-----------------------
+
+#.  Extract the Tarantool EE SDK archive:
+
+    .. code-block:: console
+
+        $ tar -xvzf tarantool-enterprise-sdk-gc64-<VERSION>-<HASH>-r<REVISION>>.linux.x86_64.tar.gz
+
+    This creates the ``tarantool-enterprise`` directory beside the archive.
+    The directory contains three executables for key Tarantool EE components:
+
+    *   ``tarantool`` -- Tarantool Enterprise Edition.
+    *   ``tt`` -- the :ref:`tt <tt-cli>` command-line utility.
+    *   ``tcm`` -- :ref:`tcm`.
+
+#.  Add the Tarantool EE components to the executable path by executing the ``env.sh``
+    script included in the distribution:
+
+    .. code-block:: console
+
+        $ source tarantool-enterprise/env.sh
+
+#.  To check that the Tarantool EE executables ``tarantool``, ``tt``,
+    and ``tcm`` are available in the system, print their versions:
+
+.. code-block:: console
+
+    $ tarantool --version
+    Tarantool Enterprise 3.0.0-0-gf58f7d82a-r23-gc64
+    Target: Linux-x86_64-RelWithDebInfo
+    Build options: cmake . -DCMAKE_INSTALL_PREFIX=/home/centos/release/sdk/tarantool/static-build/tarantool-prefix -DENABLE_BACKTRACE=TRUE
+    Compiler: GNU-9.3.1
+    C_FLAGS: -fexceptions -funwind-tables -fasynchronous-unwind-tables -static-libstdc++ -fno-common -msse2  -fmacro-prefix-map=/home/centos/release/sdk/tarantool=. -std=c11 -Wall -Wextra -Wno-gnu-alignof-expression -fno-gnu89-inline -Wno-cast-function-type -O2 -g -DNDEBUG -ggdb -O2
+    CXX_FLAGS: -fexceptions -funwind-tables -fasynchronous-unwind-tables -static-libstdc++ -fno-common -msse2  -fmacro-prefix-map=/home/centos/release/sdk/tarantool=. -std=c++11 -Wall -Wextra -Wno-invalid-offsetof -Wno-gnu-alignof-expression -Wno-cast-function-type -O2 -g -DNDEBUG -ggdb -O2
+    $ tt version
+    Tarantool CLI EE 2.1.0, linux/amd64. commit: d80c2e3
+    $ tcm version
+    1.0.0-0-gd38b12c2
+
+..  _getting_started_tcm_start:
+
+Starting TCM
+------------
+
+|tcm_full_name| is ready to run out of the box. To start it, run the following command:
+
+..  code-block:: console
+
+    $ tcm --storage.etcd.embed.enabled
+
+The :ref:`--storage.etcd.embed.enabled <tcm_configuration_reference_storage_etcd_embed>`
+option makes |tcm| start its own instance of `etcd <https://etcd.io/>`__ on bootstrap.
+This ``etcd`` instance is used for storing the :ref:`TCM configuration <tcm_configuration>`.
+
+.. note::
+
+    During the development, it is also convenient to use the TCM-embedded etcd
+    as a configuration storage for Tarantool EE clusters connected to TCM.
+    Learn more in :ref:`configuration_etcd`.
+
+.. important::
+
+    The |tcm| bootstrap log in the terminal includes a message with the credentials
```

Comment:
An important note looks a bit weird after not so important note. I'd use two steps in this section:

```
1) Start tcm
2) Find the message with creds and remember them

> not so important note
```